cob_navigation_global/launch/2dnav_linear.xml

```diff
@@ -8,7 +8,7 @@
     <remap from="odom" to="base/odometry_controller/odometry"/>
     <!-- remap outputs -->
     <remap from="cmd_vel" to="base/collision_velocity_filter/command"/>
-    <rosparam file="$(find cob_navigation_config)/$(arg robot)/linear/ctrl_params.yaml" command="load" />
+    <rosparam file="$(find cob_navigation_config)/robots/$(arg robot)/nav/linear_ctrl_params.yaml" command="load" />
```

Comment:
In order to make those launch files more easily usable in the new structure, should we add a launch arg for the config package as well?

(holds for the other launch files as well, I'm not commenting every occurance now...)